js/services/valuesService.js

```diff
@@ -1,3 +1,4 @@
 app.service('ValuesService', function(){
   this.Keyframe = dbFactory.setDb(appPath);
+  this.AppConfig = dbFactory;
```

Comment:
see note above - would think it makes sense to do the dbFactory.setAppConfigDb here